src/main/java/org/spongepowered/common/mixin/core/entity/item/EnderPearlEntityMixin.java

```diff
@@ -96,11 +97,13 @@ private boolean onEnderPearlImpact(final ServerPlayerEntity player) {
 
     /**
      * @author Zidane - June 2019 - 1.12.2
+     * @author i509VCB - Feb 2020 - 1.14.4
      * @reason Only have this ender pearl remove the thrower references if we actually changed dimension
+     * @return Change to use DimensionType
      */
     @Override
     @Nullable
-    public Entity changeDimension(final int dimensionIn) {
+    public Entity changeDimension(DimensionType dimensionIn) {
```

Comment:
Is this a vanilla method, or sponge-added method? If it's a vanilla method, and it's overriding a `@Shadow`'ed method, it should be prefixed appropriately